src/common/utils/setup-helpers.js

```diff
@@ -638,6 +639,22 @@ function maybeSetCapabilitiesForIos(options: KPOptionsObject): void {
   }
 }
 
+/**
+ * Configures airplay plugin in case we're running in Safari browser.
+ * @private
+ * @param {KPOptionsObject} options - kaltura player options
+ * @returns {void}
+ */
+function maybeConfigureAirplay(options: KPOptionsObject): void {
+  if (Env.isSafari) {
```

Comment:
Please add !Env.isSmartTV since LG2 recognize and safari browser

Reply:
so it makes more sense to check if ios or macOS then to check !smartTV in this context

Reply:
This PR is irrelevant since we will configure it via Studio